web_src/css/helpers.css

```diff
@@ -67,6 +67,7 @@ Gitea's private styles use `g-` prefix.
 .gt-self-start { align-self: flex-start !important; }
 .gt-self-end { align-self: flex-end !important; }
 .gt-no-underline { text-decoration-line: none !important; }
+.gt-line-through { text-decoration-line: line-through !important; }
```

Comment:
Can we use tailwind class directly?

Reply:
Yes, but this PR was before the tailwind PR. I will change it, as soon as possible

Reply:
> Yes, but this PR was before the tailwind PR. I will change it, as soon as possible

Yes, I see. Sorry for the inconvenience.

Reply:
Yes we should avoid addition of new helper classes now that can be covered by tailwind.